osmt_cli.sh

```diff
@@ -241,7 +241,7 @@ Usage:
   -v   Validate local environment and dependencies for development.
   -d   Start the backend Development Docker stack (MySQL, ElasticSearch, Redis). docker-compose stack will
        be detached, with containers named "osmt_dev". You can review status with 'docker ps'.
-  -e   Stop the detached backend Development Docker stack (MySQL, ElasticSearch, Redis).
+  -e   Stop & destroys the detached backend Development Docker stack (MySQL, ElasticSearch, Redis).
```

Comment:
Please remove. The Docker volumes are not destroyed. This came from a misunderstanding about stopping a Docker stack, and is not accurate. Thank you.